tools/download-test.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+while [[ $# -gt 1 ]]
```

Comment:
nit: `[[` -> `[`, and could you put the `do` on the same line (for consistency with the `if`s elsewhere)